src/extras.jl

```diff
@@ -56,14 +64,76 @@ also accept them.
 * `extend::Bool=false`: when `false`, an error is raised if some values in `x` fall
   outside of the breaks; when `true`, breaks are automatically added to include all
   values in `x`, and the upper bound is included in the last interval.
-* `labels::AbstractVector=[]`: a vector of strings giving the names to use for the
-  intervals; if empty, default labels are used.
+* `labels::Union{AbstractVector,Function}: a vector of strings giving the names to use for
+  the intervals; or a function `f(from, to, i; extend)` that generates the labels from the
```

Comment:
Should say "extend".

Maybe it would be clearer to just say `"[from, to)" or "[from, to]"` instead of giving the function?

Reply:
I agree. I just added an explanation when the closed interval is used.